internal/common/mqtt/mqttconnection_test.go

```diff
@@ -22,12 +22,46 @@ import (
 
 const Host = "broker.hivemq.com"
 const port = "1883"
+const InvalidHost = "www.broker"
```

Comment:
How about `invalid` literally?

```suggestion
const InvalidHost = "invalid"
```

Reply:
Done!